app/services/write_offer.rb

```diff
@@ -0,0 +1,28 @@
+class WriteOffer
```

Comment:
Can this be `UpdateOfferConditions` instead with `offer` and `conditions` as argument?

As offer is nested in application choice, we should be able to make any changes directly to the model as part of the Make and Change offer service, so I'm not sure if there is any value in having a dedicated service for it. 

However, as conditions do have an added complexity (specific status depending on application choice state, checking if they exist or not etc) it would make sense to have their own service.

Reply:
The idea of this service is to wrap up the logic of "I have a list of strings, please save them as conditions". Since this happens in a few places, this service feels sensible.

I imagine this service won't be required when we move to managing conditions individually though

Reply:
We have renamed the service, but I think the behaviour of it is fine unless I've misunderstood you're point